vendor/k8s.io/kubernetes/pkg/kubelet/kubelet_server_journal.go

```diff
@@ -241,7 +241,7 @@ var (
 )
 
 const (
-	dateFormat         = `2006-01-02T15:04:05.999999Z`
+	dateFormat         = `2006-01-02 15:04:05.999999`
```

Comment:
Why this change?

Reply:
Aren't you breaking the API here?

Reply:
journalctl accepts w/o those, currently if you run node-logs with ` --since "2019-12-05T08:00:00Z"` that will pass through here, but journalctl will return you error about failed parsing, from what I checked in the man page (https://www.freedesktop.org/software/systemd/man/journalctl.html): `Date specifications should be of the format "2012-10-30 18:17:16".` That's why we need this change, honestly I'm surprised this came up this late :cry:

Reply:
ok